.github/workflows/pyopenms-wheels.yml

```diff
@@ -0,0 +1,415 @@
+            for f in $Qt5_Dir/lib/*.framework/Headers
+            do
+              lnknm=$(basename "${f%/*}" ".framework")
+              ln -s $f $lnknm
+            done
+          popd
+          mkdir bld
+          pushd bld
+          # Use -DCMAKE_FIND_DEBUG_MODE=ON for debug
+          cmake -DOPENMS_CONTRIB_LIBS="$GITHUB_WORKSPACE/contribbld" -DCMAKE_PREFIX_PATH="$(echo $Qt5_Dir)/lib/cmake;${Qt5_Dir}" -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 -DBOOST_USE_STATIC=ON ../OpenMS
+          make -j4 OpenMS SuperHirn
+
+          # Search for the latest 3 main python versions in the repository pkgs/main
+          SUPPORTED_PYTHONS=$(conda search --full-name python | grep "pkgs/main" | awk '{print $2}' | cut -d '.' -f -2 | uniq | tail -3)
+
+          echo $SUPPORTED_PYTHONS
+
+          mkdir pyopenms_whls
+
+          for py in $SUPPORTED_PYTHONS
+          do
+            conda create -n pyoms-bld-$py python=$py
+            source activate pyoms-bld-$py
+
+            # set current python executable
+            CURRENT_PYTHON_EXECUTABLE=$(which python)
+
+            # pip install all the stuff
+            pip install -U setuptools
+            pip install -U pip
+            pip install -U autowrap
+            pip install -U nose
+            pip install -U numpy
+            pip install -U wheel
+
+            conda deactivate
+
+            echo $CURRENT_PYTHON_EXECUTABLE
+
+            # build pyopenms distribution (macOS)
+            cmake -DPYTHON_EXECUTABLE:FILEPATH=$CURRENT_PYTHON_EXECUTABLE -DPYOPENMS=ON .
+            make -j4 pyopenms
+
+            # copy to directory
+            cp pyOpenMS/dist/*.whl pyopenms_whls/
+
+            # clean up
+            conda remove --name pyoms-bld-$py --all
+
+          done
+
+    - uses: actions/upload-artifact@v1
+      name: Upload artifacts
+      with:
+        name: ${{ runner.os }}-wheels
+        path: bld/pyopenms_whls
+
+  build-lnx:
+    # TODO we have to make sure that this container is always updated with a push to contrib
+    runs-on: ubuntu-latest
+    container: hroest/manylinux2014_qt59_contrib:v1.1
+
+    steps:
+    # Cancels older builds if still running
+    - uses: rokroskar/workflow-run-cleanup-action@master
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master' && github.ref != 'refs/heads/develop'"
+
+    - uses: actions/checkout@v2
+      name: Checkout sources
+      with:
+        path: OpenMS
+
+    - name: Build on manylinux2014 
+      run: |
+            # Apply a patches / fixes
+            # 1. Updated readme (due to pypi.org not displaying markdown)
+            # 2. Updated version number
+            # 3. Patch init (since the linked libraries are all in .lib)
+            cd OpenMS
+            wget https://raw.githubusercontent.com/hroest/OpenMS/0f3a2e8833d18fc54b7de86e6d67ab7349e828a0/src/pyOpenMS/README.rst -O src/pyOpenMS/README.rst
```

Comment:
Can we somehow get rid of those patches?
1) E.g. just include that separate README.rst in the pyopenms directory?
2) not necessary when building from a git repo. We need to use a nightly version anyway. I am preparing a commit.
3) Figure out why boost is suddenly needed and maybe find a way to autodetermine what needs to be linked?
4) Why do we keep /OpenMS/src/pyOpenMS/pxds/SwathMapMassCorrection.pxd if it causes problems?